server/src/main/kotlin/delta/codecharacter/server/user/UserService.kt

```diff
@@ -133,6 +162,22 @@ class UserService(
         }
     }
 
+    fun verifyReCaptcha(reCaptchaResponse: String): Boolean {
+        val url =
+            "https://www.google.com/recaptcha/api/siteverify"
+        try {
+            val requestBody = ObjectMapper().writeValueAsString(mapOf("secret" to secretKey,"response" to reCaptchaResponse))
+            val client=HttpClient.newBuilder().build()
+            val request=HttpRequest.newBuilder().uri(URI.create(url)).POST(HttpRequest.BodyPublishers.ofString(requestBody)).build()
```

Comment:
```suggestion
            val request=HttpRequest.newBuilder().uri(URI.create(recaptchaUrl)).POST(HttpRequest.BodyPublishers.noBody()).build()
```
- `Request-body` should not be passed